Samples/Sample.Xamarin.Droid/Sample.Xamarin.Droid.csproj

```diff
@@ -33,6 +32,10 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <AndroidLinkMode>None</AndroidLinkMode>
+    <AotAssemblies>false</AotAssemblies>
+    <EnableLLVM>false</EnableLLVM>
+    <AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
+    <BundleAssemblies>false</BundleAssemblies>
```

Comment:
Do we expect them to be different depending on the configuration? Otherwise, instead of duplicating, can we pull them out of the condition?